libimage/manifest_list_test.go

```diff
@@ -378,6 +378,7 @@ func TestAddArtifacts(t *testing.T) {
 			err = list.AnnotateInstance(d, &aoptions)
 			require.NoError(t, err)
 
+			//nolint:usetesting
 			destination, err := os.MkdirTemp(dir, "pushed")
```

Comment:
Is there a reason why this needs to be a subdirectory of `dir` and not call the lint-expected `t.TempDir()`? I can’t immediately see it.

Reply:
That is what I had in version one and tests failed, not idea why and it didn't not produce any meaningful error output. I could try to reproduce locally but I really didn't feel like spending time on this one specific test...

Reply:
```
        manifest_list_test.go:385: TempDir: mkdir /tmp/TestAddArtifactsartifactType=applicationx-something-else,configType=applicationvnd.oci.image.config.v1+json,configLength=9,layerType=applicationvnd.oci.image.layer.v1.tar+gzip,excludeTitles=true,artifactSubject=blobs-only,indexSubject=config-only2880673117: file name too long
```

Reply:
Hah, nice. Could you note that in a comment after the //nolint to save future readers time, please?

Reply:
yes already added locally, I guess I should also file a go bug for that...